tests/unit/test_client.py

```diff
@@ -1762,6 +1762,10 @@ def test_conformance_post_policy(test_data):
                     scheme=in_data.get("scheme"),
                 )
     fields = policy["fields"]
+    out_data = test_data["policyOutput"]
+
+    decoded_policy = base64.b64decode(fields["policy"]).decode("unicode_escape")
+    assert decoded_policy == out_data["expectedDecodedPolicy"]
```

Comment:
Moved `decoded_policy` assert to the top: if something gone wrong in policy, it'll be easier to see it in decoded state - probably will save some time on debugging